pymmmc/tests/clv/models/test_beta_geo.py

```diff
@@ -0,0 +1,205 @@
+import numpy as np
+import pymc as pm
+import pytest
+import xarray as xr
+
+from pymmmc.clv.models.beta_geo import BetaGeoModel
+from pymmmc.distributions import continuous_contractual
+
+
+class TestBetaGeoModel:
+    @classmethod
+    def setup_class(cls):
+        rng = np.random.default_rng(34)
+
+        # Hyperparameters
+        a_true = 0.8
+        b_true = 2.5
+        alpha_true = 3
+        r_true = 4
+
+        # Number of subjects
+        N = 500
+        # Subject level parameters
+        p = pm.draw(pm.Beta.dist(a_true, b_true, size=N), random_seed=rng)
+        lam = pm.draw(pm.Gamma.dist(r_true, alpha_true, size=N), random_seed=rng)
```

Comment:
n = 500 is a small dataset; perhaps too small for model convergence.

I think it would make more sense to test this with the CDNOW dataset, which is a benchmarking dataset frequently used in research. This is what lifetimes uses for testing, and being able to reproduce research results is the closest thing we have to a ground truth for model assessments.

It would be worth adding a datasets utility that includes CDNOW in a separate PR.

Reply:
Resolving this in light of issue #100 and PR #104 that you just opened